src/main/scala/com.snowplowanalytics.sauna/observers/AmazonS3Observer.scala

```diff
@@ -53,7 +54,7 @@ class AmazonS3Observer(s3: S3, sqs: SQS, queue: Queue) extends Actor with Observ
 
```

Comment:
http://doc.akka.io/docs/akka/current/scala/dispatchers.html#solution-dedicated-dispatcher-for-blocking-operations